Pursuit-Core-iOS-Episodes-from-Online/Pursuit-Core-iOS-Episodes-from-Online/ViewControllers/EpisodeViewController.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  EpisodeViewController.swift
+//  Pursuit-Core-iOS-Episodes-from-Online
+//
+//  Created by albert coelho oliveira on 9/9/19.
+//  Copyright © 2019 Benjamin Stone. All rights reserved.
+//
+
+import UIKit
+
+class EpisodeViewController: UIViewController {
+    var show: ShowsWrapper?
+    @IBOutlet weak var showImage: UIImageView!
+    @IBOutlet weak var showName: UILabel!
+    @IBOutlet weak var showDescrip: UITextView!
+    @IBOutlet weak var spinner: UIActivityIndicatorView!
+    @IBOutlet weak var episodeTable: UITableView!
+    var episodes = [Episodes](){
+        didSet {
+            episodeTable.reloadData()
+            setUpLabelsShow()
+        }
+    }
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        episodeTable.delegate = self
+        episodeTable.dataSource = self
+        self.spinner.startAnimating()
+        getData()
+    }
+    func getData(){
+        Episodes.getEpisode(id: (show?.show.id)!){ (result) in
+            switch result {
+            case .failure(let error):
+                print(error)
+            case .success(let episode):
+                DispatchQueue.main.async{
+                    self.spinner.stopAnimating()
+                    return self.episodes = episode
+                }
+            }
+        }
+    }
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        guard let epVc = segue.destination as? DetailViewController else {
+            fatalError("Unexpected segue")
+        }
+        guard let selectedIndexPath = episodeTable.indexPathForSelectedRow
+            else { fatalError("No row selected") }
+        epVc.episode = episodes[selectedIndexPath.row]
+    }
+    func setUpLabelsShow(){
+        showName.text = show?.show.name
+        showDescrip.text = show?.show.fixedSummary
+        if let image = show?.show.image?.original{
```

Comment:
I like that you unwrapped the image str first before using it in the image helper, good job!